youtube_dl/downloader/common.py

```diff
@@ -55,7 +55,7 @@ class FileDownloader(object):
     Subclasses of this one must re-define the real_download method.
     """
 
-    _TEST_FILE_SIZE = 10241
+    _TEST_FILE_SIZE = 10240
```

Comment:
This was changed to be a multiple of 16 so that test content can be AES-128 decrypted. I am not aware if this breaks other extractors. I would prefer overriding it only for my extractor test, but couldn't find any sane way of doing so. Please advise.